pylib/cqlshlib/test/test_cqlsh_output.py

```diff
@@ -834,7 +834,7 @@ def test_show_output(self):
         with testrun_cqlsh(tty=True, env=self.default_env) as c:
             output = c.cmd_and_response('show version;')
             self.assertRegex(output,
-                    r'^\[cqlsh \S+ \| Cassandra \S+ \| CQL spec \S+ \| Native protocol \S+\]$')
+                    r'^\[cqlsh \S+ \| (Cassandra|Scylla) \S+ \| CQL spec \S+ \| Native protocol \S+\]$')
```

Comment:
I hope you're running the cqlsh tests against Cassandra as well, not just against Scylla?
If not, I strongly think you should, but at least try to manually confirm that the new version string works correctly in Cassandra and doesn't produce errors or references to Scylla.

Reply:
now we'll keep testing both: https://github.com/scylladb/scylla-cqlsh/pull/12